SofaKernel/modules/SofaBaseTopology/SofaBaseTopology_test/RegularGridTopology_test.cpp

```diff
@@ -207,20 +231,34 @@ std::vector<std::vector<int>> dimvalues={
     {-2,1,1, 1, 1},
 };
 
-TEST_P(RegularGridTopology_test, regularGridSize )
+TEST_P(RegularGridTopology_test, regularGridSizeComputeEdgeFromTriangle)
```

Comment:
why changing that?
the test is checking all the topology buffer sizes, not only edges.